.github/workflows/test.yml

```diff
@@ -57,3 +48,12 @@ jobs:
           yarn bundlesize
           export GITHUB_RUN_NUMBER=GITHUB_RUN_NUMBER_TMP
           echo "$GITHUB_RUN_NUMBER"
+
+      - name: Build source for remainder tests
```

Comment:
Moved tests that are dependent on built source here. Might be worth splitting out to another workflow.